openshift.spec

```diff
@@ -275,7 +411,171 @@ fi
+/etc/bash_completion.d/*
+%dir %config(noreplace) %{_sysconfdir}/origin
+%dir %config(noreplace) %{_sysconfdir}/origin/allinone/
+%dir %config(noreplace) %{_sysconfdir}/origin/allinone/master/
+%dir %config(noreplace) %{_sysconfdir}/origin/allinone/node/
+
+%files -n atomic-enterprise-master
+%defattr(-,root,root,-)
+%{_unitdir}/atomic-enterprise-master.service
+%{_unitdir}/atomic-enterprise-allinone.service
+%config(noreplace) %{_sysconfdir}/sysconfig/atomic-enterprise-master
+%config(noreplace) %{_sysconfdir}/sysconfig/atomic-enterprise-allinone
+%dir %config(noreplace) %{_sysconfdir}/origin/master
+%dir %config(noreplace) %{_sysconfdir}/origin/allinone/master
+%dir %config(noreplace) %{_sysconfdir}/origin/allinone/node
+%ghost %config(noreplace) %{_sysconfdir}/sysconfig/%{name}-master
+%ghost %config(noreplace) %{_sysconfdir}/origin/master
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/admin.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/admin.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/admin.kubeconfig
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/ca.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/ca.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/ca.serial.txt
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/etcd.server.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/etcd.server.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/master-config.yaml
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/master.etcd-client.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/master.etcd-client.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/master.kubelet-client.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/master.kubelet-client.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/master.server.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/master.server.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/openshift-master.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/openshift-master.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/openshift-master.kubeconfig
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/openshift-registry.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/openshift-registry.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/openshift-registry.kubeconfig
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/openshift-router.crt
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/openshift-router.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/openshift-router.kubeconfig
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/policy.json
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/serviceaccounts.private.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/master/serviceaccounts.public.key
+%ghost %config(noreplace) %{_sysconfdir}/origin/allinone/master/admin.crt
```

Comment:
Aren't `/etc/origin/allinone/` config files supposed to be actually present in the RPM? I'm referring to the `%ghost` directive.